level2/securitygroup.tf

```diff
@@ -0,0 +1,60 @@
+# Security group for the main-ec2 instance to allow ssh inbound on port 22
+resource "aws_security_group" "allow_ssh" {
+  name        = "ec2-instance"
+  description = "Allow SSH inbound traffic"
+  vpc_id      = data.terraform_remote_state.main-vpc.outputs.main-vpc-id
+
+  ingress {
+    description = "ssh to VPC"
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["70.95.138.224/32"]
+
+  }
+  ingress {
+    description = "http to VPC"
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
You don't need to open 80 everywhere in this security group. Instead, you can allow connections on port 80 from the security group of your load balancer

Reply:
removed the port 80 inbound rule and added the load balancer security group to the ec2 instance